app/Http/Controllers/EventController.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Storage;
+use App\Models\Event;
+use App\Models\Level;
+
+class EventController extends Controller
+{
+    const LOCAL_STORAGE_FOLDER = 'public/images/';
+    private $event;
+    private $level;
+
+    
+
+    public function __construct(Event $event, Level $level){
+        $this->event = $event;
+        $this->level = $level;
+    }
+
+    public function show(){
```

Comment:
rename this to `create()`

Reply:
once change please also update your `web.php`

Reply:
This method, I would like to show all the event to users which admin had created.   Is it okay to name like "index()" or "all_events()" instead of "create()"?

Reply:
yes, if you have an index function yet then it's okay